tests/test_wave_05.py

```diff
@@ -57,7 +57,7 @@ def test_new_genre_rec_from_empty_friends():
     # ****** Complete the Act and Assert Portions of theis tests **********
     # *********************************************************************
```

Comment:
For this test we need to complete both the act & assert steps. What would that look like?